app.yaml

```diff
@@ -32,6 +32,7 @@ handlers:
 - url: /build
   static_dir: build
   secure: always
+  application_readable: true
```

Comment:
The _application_readable_ is needed for the gadgets/RTEs/interactions _html_body_ function to work, because this function loads HTMLs in backend.